src/main/java/programminglife/ProgrammingLife.java

```diff
@@ -28,36 +32,37 @@
     private static final String DATA_FOLDER = "data/";
     private static final String TB_DATA = DATA_FOLDER + "real/TB10.gfa";
     private static final String HUMAN_DATA = DATA_FOLDER + "real/chr19.hg38.w115.gfa";
-
     private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("m:ss.SSS", Locale.getDefault());
 
+    /**
+     * Main method for the application.
+     * @param args argument
+     */
     public static void main(String[] args) {
-        Graph g;
-        String graphFile;
-        DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
-
-        if (args.length > 0) {
-            graphFile = args[0];
-        } else {
-            System.err.println("No GFA file specified!");
-            System.exit(1);
-            return; // This is to make sure the Java compiler does not warn about uninitialized 'graphFile'
-        }
-
-        try {
-            long startTime = System.nanoTime();
-            g = Graph.parse(graphFile);
-            long elapsedTimeMs = (System.nanoTime() - startTime) / 1000000;
-            System.out.printf("Parsing took %s\n", DATE_FORMAT.format(elapsedTimeMs));
-        } catch (FileNotFoundException e) {
-            System.err.printf("File not found (%s)\n", graphFile);
-        } catch (UnknownTypeException e) {
-            System.err.printf("Parse error: %s\n", e.getMessage());
-        }
-
-//        launch(args);
-
-        Platform.exit();
+//        Graph g;
+//        String graphFile;
+//        DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
+//
+//        if (args.length > 0) {
+//            graphFile = args[0];
+//        } else {
+//            System.err.println("No GFA file specified!");
+//            System.exit(1);
+//            return; // This is to make sure the Java compiler does not warn about uninitialized 'graphFile'
+//        }
+//
+//        try {
+//            long startTime = System.nanoTime();
+//            g = Graph.parse(graphFile);
+//            long elapsedTimeMs = (System.nanoTime() - startTime) / 1000000;
+//            System.out.printf("Parsing took %s\n", DATE_FORMAT.format(elapsedTimeMs));
+//        } catch (FileNotFoundException e) {
+//            System.err.printf("File not found (%s)\n", graphFile);
+//        } catch (UnknownTypeException e) {
+//            System.err.printf("Parse error: %s\n", e.getMessage());
+//        }
+
```

Comment:
I left it there as it might be needed to update the parsing in other locations and can still be removed from there